vspace/src/main/webapp/WEB-INF/views/staff/spaces/space.html

```diff
@@ -98,28 +135,111 @@
 		});
 	[/]
 	[/]
+	
+	[# th:each="link,loop : ${spaceLinks}"]

```

Comment:
didn't we discuss replacing these thymeleaf loops with javascript loop that load the links asynchronically? To get cleaner javascript.